packages/meditrak-app-server/src/routes/sync/getChangesFilter.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { getHighestPossibleIdForGivenTime } from '@tupaia/database';
+import { DbFilter } from '@tupaia/server-boilerplate';
+import { getSupportedModels } from '../../sync';
+import { MeditrakSyncQueueFields } from '../../models/MeditrakSyncQueue';
```

Comment:
All this is copied from https://github.com/beyondessential/tupaia/blob/973ac52ae0381ad668e7f417f664ee7fb63b2927/packages/central-server/src/apiV2/utilities/getChangesFilter.js